app/views/shared/_hot_items.html.erb

```diff
@@ -1,287 +1,27 @@
-              <a href="#" class="btn btn-default add-to-cart"><i class="fa fa-shopping-cart"></i>Add to cart</a>
-            </div>
-
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-3">
-        <div class="product-image-wrapper">
-          <div class="single-products">
-            <div class="productinfo text-center">
-              <img src="images/home/gallery4.jpg" alt="" />
-              <h2>$56</h2>
-              <p>Easy Polo Black Edition</p>
-              <a href="#" class="btn btn-default add-to-cart"><i class="fa fa-shopping-cart"></i>Add to cart</a>
-            </div>
-
-          </div>
-        </div>
-      </div>
-    </div>
-
-    <div class="tab-pane fade" id="poloshirt">
-      <div class="col-sm-3">
-        <div class="product-image-wrapper">
-          <div class="single-products">
-            <div class="productinfo text-center">
-              <img src="images/home/gallery2.jpg" alt="" />
-              <h2>$56</h2>
-              <p>Easy Polo Black Edition</p>
-              <a href="#" class="btn btn-default add-to-cart"><i class="fa fa-shopping-cart"></i>Add to cart</a>
-            </div>
-
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-3">
-        <div class="product-image-wrapper">
-          <div class="single-products">
-            <div class="productinfo text-center">
-              <img src="images/home/gallery4.jpg" alt="" />
-              <h2>$56</h2>
-              <p>Easy Polo Black Edition</p>
-              <a href="#" class="btn btn-default add-to-cart"><i class="fa fa-shopping-cart"></i>Add to cart</a>
-            </div>
-
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-3">
-        <div class="product-image-wrapper">
-          <div class="single-products">
-            <div class="productinfo text-center">
-              <img src="images/home/gallery3.jpg" alt="" />
-              <h2>$56</h2>
-              <p>Easy Polo Black Edition</p>
-              <a href="#" class="btn btn-default add-to-cart"><i class="fa fa-shopping-cart"></i>Add to cart</a>
-            </div>
-
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-3">
-        <div class="product-image-wrapper">
-          <div class="single-products">
-            <div class="productinfo text-center">
-              <img src="images/home/gallery1.jpg" alt="" />
-              <h2>$56</h2>
-              <p>Easy Polo Black Edition</p>
-              <a href="#" class="btn btn-default add-to-cart"><i class="fa fa-shopping-cart"></i>Add to cart</a>
-            </div>
-
-          </div>
-        </div>
+<div class="recommended_items">
+  <!--recommended_items-->
+  <h2 class="title text-center"><%= t("label.all_items") %></h2>
+
+  <div id="recommended-item-carousel" class="carousel slide" data-ride="carousel">
+    <div class="carousel-inner">
+      <div class="item active">
+        <% for i in 0..(@trend_items.length-3) do %>
```

Comment:
sao ko each mà dùng for em?

Reply:
Cái này là do cái layout của nó lúc đầu cho hiện 3 cái items thôi chị. 3 items đầu tiên hiện thị ra ngoài, nó thuộc class active. Những item sau thì nó ở class khác. :( Em dùng for để xử lý cho dễ. Vì em vẫn chưa biết  nên xử front end đoạn này, nếu each thì nó lấy hết ra, bay mất layout chị ơi.